app/Models/Store/Product.php

```diff
@@ -100,6 +103,12 @@ public function getDescriptionAttribute($value)
         }
     }
 
+    public function isAvailable() : bool
+    {
+        return $this->enabled
+            && $this->available_until === null ? true : $this->available_until->isFuture();
```

Comment:
should be put inside (brackets) otherwise explosions when not enabled and null `available_until`. Or `optional(x)->a() ?? default`.